WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -558,6 +556,14 @@ import WordPressComAnalytics
             presentViewController(alertController, animated: true, completion: nil)
         }
     }
+    
+    private func sharePost(postID: NSManagedObjectID, fromView anchorView: UIView) {
+        if let post = self.postWithObjectID(postID) {
```

Comment:
Nitpicky, I know, but maybe use a `guard` instead and exit early vs doing the work inside the block.